### ISSUE TITLE
About the loss

### ISSUE THREAD

**Issue by SherlockHolmes221** [IDEA-opensource/DN-DETR#21]
Where is the reconstruction loss? I can not find the loss in DABDETR.py? Thanks

**Comment by FengLi-ust** [comment]
We wrap all dn components in `dn_components.py`, including the ` compute_dn_loss` function.
We calculate losss with `dn_losses = compute_dn_loss(mask_dict, self.training, aux_num, self.focal_alpha)` in DABDETR.py. Thank you.

**Comment by SherlockHolmes221** [comment]
Thank you for your reply.
